src/homepageExperience/utils.ts

```diff
@@ -34,3 +34,34 @@ export const HOMEPAGE_NAVIGATION_STEPS = [
     glyph: IconFont.StarSmile,
   },
 ]
+
+export const HOMEPAGE_NAVIGATION_STEPS_CLI = [
+  {
+    name: 'Overview',
+    glyph: IconFont.BookOutline,
+  },
+  {
+    name: 'Install \n Dependencies',
+    glyph: IconFont.Install,
+  },
+  {
+    name: 'Initialize \n Client',
+    glyph: IconFont.FileSettings,
+  },
+  {
+    name: 'Write \n Data',
+    glyph: IconFont.Pencil,
+  },
+  {
+    name: 'Execute \n Flux Query',
+    glyph: IconFont.PlayOutline,
+  },
+  {
+    name: 'Execute \n Aggregate',
+    glyph: IconFont.PlayOutline,
+  },
```

Comment:
let's remove the spaces around the `\n` characters, since the spaces don't need to be there, and if someone highlights either `Execute` or `Aggregate`, `Execute \n Aggregate`, and copy/pastes, they'll get an extra whitespace character.